include/ostro-xt-supported-recipes.txt

```diff
@@ -0,0 +1,3 @@
+libevent@core
+libxscrnsaver@core
+nss@core
```

Comment:
> Contents in this file depend highly on what's included in the default Ostro OS XT image and user configuration in local.conf.

As the README.md stated, this file is only useful for developers to build with default Ostro OS XT image.

> How are you going to maintain the list to keep it up-to-date and valid for common configurations?

I understand "up-to-date" here means keep it updated to Ostro OS XT change. I think we need buildbot setup to track both `meta-crosswalk` and `ostro-os-xt` for potential breaks.